tests/unit/transformers/test_id.py

```diff
@@ -591,6 +591,8 @@ def test__reverse_transform_info_message(self, mock_logger):
         """
         # Setup
         instance = RegexGenerator('[A-Z]', cardinality_rule=None)
+        del instance.cardinality_rule  # coverage and backwards compatibility
```

Comment:
why do we have to delete this?

Reply:
Ah, can we just add that to the test description?

Reply:
Done in [6cf6dae](https://github.com/sdv-dev/RDT/pull/957/commits/6cf6daead164fb4d4cf570f9035467b6a6f1e8f9)